### PR TITLE
How to return data in XML format

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3883](https://openedx.atlassian.net/browse/DOC-3883)

Added a section on how to get your return data in XML format.

### Date Needed 

This feature is in production now, so getting the docs updated soon would be good.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @georgebabey 
- [x] Subject matter expert: @asadiqbal08 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review: 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Squash commits

### CODE REVIEW

File: en_us/enterprise_api/source/api_reference/reference.rst [hunk 1]
```diff
@@ -45,6 +45,25 @@ The following endpoints are available in the Enterprise API.
...
   course run. For details, see :ref:`course_enrollments Endpoint`.
 
+.. _Returning XML Data:
+
+************************************
+Returning Data in XML or JSON Format
+************************************
+
+By default, the edX Enterprise API returns data in JSON format. It can also,
+if you choose, return data in XML format. To specify that return data should
```
Comment (srpearce) [inline en_us/enterprise_api/source/api_reference/reference.rst]: Strike "if you choose"

File: en_us/enterprise_api/source/api_reference/reference.rst [hunk 2]
```diff
@@ -45,6 +45,25 @@ The following endpoints are available in the Enterprise API.
...
+use XML format, include the ``Accept: appication/xml`` header in your API
+request. For example, to request XML-formatted information about a course run
+using ``curl``, send a request similar to the following.
```
Comment (srpearce) [inline en_us/enterprise_api/source/api_reference/reference.rst]: We try to include nouns after "following" in our examples

Comment (asadiqbal08) [reply en_us/enterprise_api/source/api_reference/reference.rst]: data will also be available in form of JSON / XML if user add the .json / .xml extensions in the api endpoint.

**Review by georgebabey — APPROVED** [review]
This looks great. Thanks @grantgoodmanedX

**Pull request merged**